controllers/cloud.go

```diff
@@ -29,6 +29,7 @@ import (
 var (
 	instanceStateTagKey = "upgrademgr.keikoproj.io/state"
 	inProgressTagValue  = "in-progress"
+	completedTagValue   = "completed"
```

Comment:
I don't think we need to set this completed tag.
I think what happens is when we start working on an instance we tag with the `in-progress` tag, and then we know to pick those up first if we resume, however once we terminate, the selector ignores terminated instances, hence we don't really need to make another API call to modify the tag to completed.

Reply:
I agree. This was before I started filtering out instances based on states.